### PR TITLE
Replace postcss-custom-properties-fallback with local-version

### PR DESCRIPTION
**Pull request by nielslyngsoe**
Replacing the plugin with local version to include bugfix, as plugin is not maintained.
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

- [x] If my change required a change to the documentation, I have updated the documentation in this pull request.
- [x] I have read the **[CONTRIBUTING](<(https://github.com/umbraco/Umbraco.UI/blob/dev/docs/CONTRIBUTING.md)>)** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by iOvergaard** [comment]
This also fixes AB#15528 to avoid having a duplicate CommonJS and es module for custom properties and the lit css fallback plugin.

File: package.json [hunk 1]
```diff
@@ -25,8 +25,8 @@
     "format:prettier": "prettier \"**/*.js\" \"**/*.ts\" \"**/*.mjs\"  --write --ignore-path .gitignore",
     "lint": "npm run lint:eslint && npm run lint:prettier",
     "format": "npm run format:eslint && npm run format:prettier",
-    "storybook": "npm run bootstrap && npm run storybook:analyze && start-storybook -p 6006 -s ./.storybook/images",
```
Comment (nielslyngsoe) [inline package.json:28]: @iOvergaard what is the reason to remove "-s ./.storybook/images"?

Comment (iOvergaard) [reply package.json:28]: @nielslyngsoe It is deprecated and instead you set up staticDirs in main.js

Comment (nielslyngsoe) [reply package.json:28]: Perfect, thanks for the explanation.

**Pull request merged**